backend/api/Areas/Leases/Mapping/Lease/SecurityDepositMap.cs

```diff
@@ -17,7 +17,6 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.SecurityDepositType, src => src.SecurityDepositTypeCodeNavigation.Description)
                 .Map(dest => dest.Description, src => src.Description)
                 .Map(dest => dest.AmountPaid, src => src.AmountPaid)
-                .Map(dest => dest.TotalAmount, src => src.TotalAmount)
```

Comment:
@asanchezr this appears unused in deposits but want to confirm.

Reply:
yes - this is no longer stored on DB. It is calculated from amount paid and interest to date